### ISSUE TITLE
ImportError: cannot import name force_managed

### ISSUE THREAD

**Issue by sametmax** [MnogoByte/django-permanent#1]
```
      2 from operator import attrgetter
      3 
----> 4 from django.db.models.deletion import Collector as BaseCollector, force_managed
      5 from django.db.models import signals, sql
      6 from django.utils import six

ImportError: cannot import name force_managed
```

Using django 1.6. I grepped the django code and can't find force_managed anywhere, so I'm guessing it's been removed.

**Comment by meteozond** [comment]
Yep! 1.6 brings significant transactions and querysets backward incompatible changes.
And force_managed not only one problem.
A lot of changes and tests have to be done. It would be major version.
Unfortunately I'm not ready to do it right now.
I hope to implement it in the nearest future.
Suppose 1.5.\*  is temporary the strict requirement for this package.
Sorry.

**Comment by meteozond** [comment]
Hi, @sametmax I've got two news - good and bad.
Good one - I've implemented 1.6 features and made tests passing in the 1.6 test environment.
Bad - Unfortunately, I haven't got any 1.6 projects in development at the moment. 
Won't you mind to test latest version in some 1.6 environment?
https://github.com/meteozond/django-permanent/archive/1.0.0.zip
